README.md

```diff
@@ -10,7 +10,7 @@ apply plugin: 'checkstyle'
 
     checkstyle {
         config project.resources.text.fromUri('https://raw.githubusercontent.com/oyewaleoyelami/check/master/checkstyle.xml')
-        toolVersion = "8.11"
+        toolVersion = "6.0"
```

Comment:
Why?

Reply:
trying to ascertain the classes and modules available under each versions as few were deprecated already.. changed to 8.12 now which is the latest